### ISSUE TITLE
High Priority - WCAG Level A - Non-text content on Members Listing page have repetitive alt attributes

### ISSUE THREAD

**Issue by JuliaSang** [w3c/w3c-website#164]
Images were present on the page which have been provided with descriptive alt attributes that are repetitive of the link text which is adjacent to them.

**WCAG Reference:**
1.1.1 Non-text Content (Level A)
[Understanding Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) | [How to Meet Non-text Content](https://www.w3.org/WAI/WCAG21/quickref/#non-text-content)

**Issue ID: DAC_Non_text_content_01**

URL: https://beta.w3.org/membership/list/ 
Page title: Current Members & Testimonials 
Journey 2, Step 1
<img width="775" alt="member listing" src="https://user-images.githubusercontent.com/20167528/159247094-6cabded1-9e1f-4abf-b8f0-5db0879e6d7b.png">

Images are present that have been provided with a descriptive alt attribute, for example alt="51Degress", which is identical to the link text of the heading which is adjacent to the image meaning screen reader users will hear this information twice.

As the image does not provide screen reader users with additional meaningful information it is considered decorative. Decorative images can often be confusing or time-consuming for screen readers to navigate past especially if there are multiple images on the page.

**Current code ref(s):** #main > div.member-list > div:nth-child(2) > div > div.sidebar > div > img

```
<div class="l-sidebar card card--member" data-component="card"> 
<div>
<div class="not-sidebar card__text">
<h2 class="card__heading"><a class="card__link"  								href="https://51degrees.com">51Degrees</a></h2> 
<p>51Degrees provides a one-stop-shop for real-time data services, with open source APIs for device detection, geolocation, and more. We are members of W3C to protect the vision of One Web for the sake of users and organizations all around the world.
</p>
</div>
<div class="sidebar">
<div class="logo--member">
<img src="https://cdn.w3.org/thumbnails/120/logos/organizations/122675.png?x-version=3" alt="51Degrees" loading="lazy" />  								      </div>     
</div>
</div> 
</div>
```

**Solution:**
It is recommended that where images do not convey meaningful information to screen reader users, that they be concealed appropriately; this can be achieved by providing the image with a null alt attribute (alt="") which will enable screen reading software to disregard the image entirely. This applies to all of the images in the member list cards on this page.

Please see F38: [Failure of Success Criterion 1.1.1 due to not marking up decorative images in HTML in a way that allows assistive technology to ignore them](https://www.w3.org/TR/WCAG20-TECHS/F38.html) for more information.

**Example:**

```
<div class="l-sidebar card card--member" data-component="card">
</div> 
<div class="not-sidebar card__text">
<h2 class="card__heading"><a class="card__link" href="https://51degrees.com">51Degrees</a></h2>  								<p> 51Degrees provides a one-stop-shop for real-time data services, with open source APIs for device detection, geolocation, and more. We are members of W3C to protect the vision of One Web for the sake of users and organizations all around the world. </p>
</div>     <div class="sidebar"> <div class="logo--member">         
<img src="https://cdn.w3.org/thumbnails/120/logos/organizations/122675.png?x-version=3" alt="" loading="lazy" />  								      </div> 
</div> 
</div> 
</div>

```

**Comment by vivienlacourba** [comment]
Fixed by @deniak in https://gitlab.w3.org/symfony-bundles/main/-/merge_requests/180
